hw1/arraySort/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+class Program
+{
+    static void Swap(ref int first, ref int second)
```

Comment:
```suggestion
    private static void Swap(ref int first, ref int second)
```
По традиции модификаторы видимости пишут всегда, даже если умолчания устраивают.